<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PermissionDialog</class>
 <widget class="QDialog" name="PermissionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>488</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>授权管理</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/zansan.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a2ff;&quot;&gt;试用期结束后，raw格式&lt;/span&gt;&lt;span style=&quot; color:#c1c1c1;&quot;&gt;(cr2,cr3,dng等格式)&lt;/span&gt;&lt;span style=&quot; color:#a8a2ff;&quot;&gt;转换需要激活后才可使用！其余格式转换无需激活授权依旧可以使用！&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;激活步骤：&lt;/span&gt;&lt;span style=&quot; color:#989898;&quot;&gt;1. 扫描下方二维码支付20元；2. 添加微信“FinnSoft”为好友；3. 点击“复制”，复制注册码，并发送给FinnSoft；4. FinnSoft生成激活码后，会发送给您；5. 复制FinnSoft发送给您的激活码，粘贴到上面的激活码输入框中；6. 点击“激活”，即可完成！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>注册码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditReg">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCopyReg">
      <property name="text">
       <string>复制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>激活码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditActivate"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnActivate">
      <property name="text">
       <string>激活</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
